PCLConfig.cmake.in

```diff
@@ -93,19 +93,13 @@ macro(find_boost)
   elseif(NOT BOOST_INCLUDEDIR)
     set(BOOST_INCLUDEDIR "@Boost_INCLUDE_DIR@")
   endif()
-  # use static Boost in Windows
-  if(WIN32)
-    set(Boost_USE_STATIC_LIBS @Boost_USE_STATIC_LIBS@)
-    set(Boost_USE_STATIC @Boost_USE_STATIC@)
-    set(Boost_USE_MULTITHREAD @Boost_USE_MULTITHREAD@)
-  endif()
+  
   set(Boost_ADDITIONAL_VERSIONS
     "@Boost_MAJOR_VERSION@.@Boost_MINOR_VERSION@.@Boost_SUBMINOR_VERSION@" "@Boost_MAJOR_VERSION@.@Boost_MINOR_VERSION@"
     "1.76.0" "1.76" "1.75.0" "1.75" 
     "1.74.0" "1.74" "1.73.0" "1.73" "1.72.0" "1.72" "1.71.0" "1.71" "1.70.0" "1.70"
     "1.69.0" "1.69" "1.68.0" "1.68" "1.67.0" "1.67" "1.66.0" "1.66" "1.65.1" "1.65.0" "1.65")
-  # Disable the config mode of find_package(Boost)
-  set(Boost_NO_BOOST_CMAKE ON)
+  
   find_package(Boost 1.65.0 ${QUIET_} COMPONENTS @PCLCONFIG_AVAILABLE_BOOST_MODULES@)
```

Comment:
The hard-coding of 1.65 seems suspicious

Reply:
Its just the minimum required version, ie. equal to what we have on CI's right?

We have the same for Eigen and FLANN:
https://github.com/PointCloudLibrary/pcl/blob/7f87657e8f796373852ff7f80228a1573239de8d/CMakeLists.txt#L302-L307